package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "amoeba",
-  "version": "0.4.7",
+  "version": "0.6.0",
```

```suggestion
  "version": "0.5.0",
```
Unless I missed another reason to skip `0.5.0`...